src/main/java/com/amazonaws/encryptionsdk/kmsv2/KmsMasterKeyProvider.java

```diff
@@ -127,83 +108,27 @@ private RuntimeException clientSupplierComboException() {
               + "withCredentials or withClientBuilder");
```

Comment:
If we really can't support withCredentials in SDKv2, then lets do a grep and make sure we remove every reference to it.

Reply:
good call! I'll double check everything for credentials references